stockroom/storages/paramstore.py

```diff
@@ -1,2 +1,11 @@
-class ParamStore:
-    pass
\ No newline at end of file
+from .storagebase import StorageBase
+
+
+class ParamStore(StorageBase):
+    def __init__(self, write):
+        self.write = write
+        super().__init__()
+
+
+def paramstore(write=False):
+    return ParamStore(write=write)
```

Comment:
Copy / paste of my comment on the `datastore` module

> I'm just going to suggest that for right now, if you don't have something implemented for a storage type, you don't include the module at all.
>
> Since this does nothing, there's no documentation, and nothing else (should) depend on it, there's no point in keeping a placeholder module around. 
> 
> I'm going to leave this pycon talk here instead of writing a wall of text explaining why; https://www.youtube.com/watch?v=o9pEzgHorH0